exe-common/Cargo.toml

```diff
@@ -2,6 +2,14 @@
 name = "exe-common"
 version = "0.1.0"
 authors = ["Vincent Hanquez <vincent.hanquez@iohk.io>", "Nicolas Di Prima <nicolas.diprima@iohk.io>", "The Rust-Crypto Project Developers"]
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/input-output-hk/rust-cardano"
+homepage = "https://github.com/input-output-hk/rust-cardano/exe-common#README.md"
+description = """
+Helpers and toolings for the cardano protocol, keeping blockchain synced and storing
+the blockchain locally.
```

Comment:
Helpers and tools for the Cardano protocol. This includes keeping the blockchain synced and storing the blockchain locally.